coral-component-shell/src/scripts/ShellSelectListSwitcherItem.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {BaseComponent} from '../../../coral-base-component';
+import {transform} from '../../../coral-utils';
+
+const CLASSNAME = '_coral-Shell-selectlistswitcher-item';
+
+/**
+ @class Coral.Shell.SelectListSwitcherItem
+ @classdesc A Shell SelectListSwitcherItem component
+ @htmltag coral-shell-selectlistswitcher-item
+ @extends {HTMLElement}
+ @extends {BaseComponent}
+ */
+class ShellSelectListSwitcherItem extends BaseComponent(HTMLElement) {
+
+/**
+   href property of item
+   @type {String}
+   @htmlattribute href
+   @htmlattributereflected
+   */
+  get href() {
+    return this._href;
+  }
+
+  set href(value) {
+    var href = transform.string(value);
+    var update = false;
+    if(this._href) {
+      update = true;
```

Comment:
Can't seem to understand the use of update variable, we are only triggerring chnage when item is true

Reply:
for the first time when the href is set, this._href will be undefined thus the update event won't be triggered. This will work only when the already set href is changed.